docs/src/using.md

```diff
@@ -102,15 +102,17 @@ levels
 levels!
 ```
 
-## Handling Missing Values: NullableCategoricalArray
+## Handling Missing Values
 
-The examples above assumed that the data contained no missing values. This is generally not the case in real data. This is where `NullableCategoricalArray` comes into play. It is essentially the categorical-data equivalent of [NullableArrays](https://github.com/JuliaStats/NullableArrays.jl). It behaves exactly the same as `CategoricalArray` , except that it returns `Nullable{CategoricalValue}` elements when indexed. See [the Julia manual](http://docs.julialang.org/en/stable/manual/types/?highlight=nullable#nullable-types-representing-missing-values) for more information on the `Nullable` type.
+The examples above assumed that the data contained no missing values. This is generally not the case in real data. This is where `CategoricalArray{?T}` comes into play. It is essentially the categorical-data equivalent of `Array{?T}`. It behaves exactly the same as `CategoricalArray{T}` , except that when indexed it returns either a `CategoricalValue{T}`, or `null` if a value is missing. See [the Nulls package](https://github.com/JuliaData/Nulls.jl) for more information on the `Null` type.
 
-Let's adapt the example developed above to support missing values. At first sight, not much changes: 
+Let's adapt the example developed above to support missing values. Since there are no missing values in the input vector, we need to specify that the array should be able to hold eithera `String` or `null`. Here, `?String` is a shorthand for `Union{Null, String}`:
```

Comment:
@quinnj It would be great if you could fix this one for me at the same time as the indentation issues I noted. BTW, I guess `?String` should be removed just like you did in the rest of the code?